code/mob/living.dm

```diff
@@ -682,7 +682,7 @@
 			return src.eyecam.client.preferences.auto_capitalization
 	. = ..()
 
-/mob/living/say(var/message, ignore_stamina_winded)
+/mob/living/say(var/message, ignore_stamina_winded, unique_maptext_style, maptext_animation_colors)
```

Comment:
```suggestion
/mob/living/say(var/message, ignore_stamina_winded, var/unique_maptext_style = null, var/maptext_animation_colors = null)
```
Gotta give these arguments default values, otherwise the linter is unhappy cuz there are places in other parts of the code where it's just `mob.say("asjfkl")` and there are no values provided for `unique_maptext_style` and `maptext_animation_colors`.

Reply:
Makes a lot of sense, changed! Though as far as I can tell the linter is still upset about it

Reply:
Also makes sense! and fixed, which seems to have placated the linter